docs/user_guide.md

```diff
@@ -252,6 +256,77 @@ The images can be filtered by labels:
+1. Click **Add** at the bottom of the window to add the CVEs to the project whitelist.
+1. Optionally uncheck the **Never expires** checkbox and use the calendar selector to set an expiry date for the whitelist.
+1. Click **Save** at the bottom of the page to save your settings.
+
+After you have created a project whitelist, you can remove CVE IDs from the list by clicking the delete button next to it in the list. You can click **Add** at any time to add more CVE IDs to the whitelist for this project. 
+
+If CVEs are added to the system whitelist after you have created a project whitelist, click **Add System** to add the new entries from the system whitelist to the project whitelist. 
+
+**NOTE**: If CVEs are deleted from the system whitelist after you have created a project whitelist, and if you added the system whitelist to the project whitelist, you must manually remove the deleted CVEs from the project whitelist. If you click **Add System** after CVEs have been deleted from the system whitelist, the deleted CVEs are not automatically removed from the project whitelist.
+
+## Set Project Quotas
+
+To exercise control over resource use, as a system administrator you can set  quotas on projects. You can limit the number of artifacts that a project can contain and limit the amount of storage capacity that a project can consume. You can set default quotas that apply to all projects globally. You can also set quotas on individual projects. If you set global default quotas and also set different quotas on individual projects, the per-project quotas override the global default quotas.
+
+By default, all projects have unlimited quotas for both artifacts and storage use. 
+
+1. Go to **Configuration** > **Project Quotas**.
+   ![Project quotas](img/project-quota1.png)
+1. To set global default quotas on all projects, click **Edit**.
+   ![Project quotas](img/project-quota2.png)
+   1. For **Default artifact count**, enter the maximum number of artifacts that any project can contain, or enter `-1` to set the default to unlimited. 
+   1. For **Default storage consumption**, enter the maximum quantity of storage that any project can consume, selecting `MB`, `GB`, or `TB` from the drop-down menu.
+   ![Project quotas](img/project-quota3.png)
+   1. Click **OK**.
+1. To set quotas on an individual project, click the 3 vertical dots next to a project and select **Edit**.
+   ![Project quotas](img/project-quota4.png)
+   1. For **Default artifact count**, enter the maximum number of artifacts that this individual project can contain, or enter `-1` to set the default to unlimited. 
+   1. For **Default storage consumption**, enter the maximum quantity of storage that this individual project can consume, selecting `MB`, `GB`, or `TB` from the drop-down menu.
+
+After you set quotas, the you can see how much of their quotas each project has consumed in the **Project Quotas** tab.
+
+![Project quotas](img/project-quota5.png)

```

Comment:
Some additional info that needs to notify user,

[Background]
1, Harbor computes size on putting blob/manifest from docker client
2, Harbor computes count on putting manifest from docker client.
3, Shared blob will be only computed once per project. In docker, the definition of blob sharing is globally, but in Harbor, we narrow it down to the project.


1, For `Retag`, it will also book/release resources. 

-  Retag image A from project A to project A, with a different tag. The artifact count increase one, but disk do not change as no more new blobs/manifest introduced.
-  Retag image A from project A to project B. The artifact count and disk are updated accordingly.

2, For `GC`, docker push will put the config/blobs firstly, and finally put the manifest.

- If the count reaches the limit, the blobs of the image can be pushed into the project and the storage usage will be updated accordingly. We can call these blob as `untagged blobs`, and these can be removed by GC, and the storage usage will be returned after GC.

3, The overall storage may larger than the actual disk capacity as mentioned in background #3.

4, Chart size is not calculated(Only for artifact count)

Reply:
Added the above information in https://github.com/goharbor/harbor/pull/8869/commits/78c7238cdfa7fec16200db9fd9827fd6f07d9436.

Reply:
@wy65701436 can you please review https://github.com/goharbor/harbor/commit/78c7238cdfa7fec16200db9fd9827fd6f07d9436 ? Thanks!

Reply:
@xaleeks has looked at this section since the update, so I think that we can resolve this one.